src/lines_to_params.rs

```diff
@@ -0,0 +1,165 @@
+use crate::parser;
+use pyo3::prelude::*;
+use pyo3::types::PyDict;
+
+type ActionEffectParams = (u8, u8, u8, u8, u16, u8, u8);
+type StatusEffectParams = (u16, u16, f32, u32);
+
+type StatuslistParams<'a> = (
+    (u32, &'a str),
+    &'a str,
+    Vec<u32>,
+    Vec<f32>,
+    Vec<(u16, u16, f32, u32)>,
+);
+
+/// Get id name type from tuple
+#[pyfunction]
+#[pyo3(text_signature = "(name_id_pair: list[str]) -> (int, str)")]
+pub(crate) fn parse_id_name_pair(inp: Vec<&str>) -> (u32, &str) {
```

Comment:
If this isn't exposed, why is it a pyo3 function?

Reply:
It's because I just moved the old function and never removed it.